src/db/main.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const { Sequelize } = require('sequelize');
+
+const sequelize = new Sequelize('postgres', 'postgres', 'brambllee', {
```

Comment:
It's best practice to avoid hardcoding database credentials. Consider using environment variables for storing sensitive information like username, password, and host 🛡️